### PR TITLE
ENH/STYLE: contrib brl, bpgl & bsgm

### PR DESCRIPTION
**Pull request by drewgilliam**
## PR Checklist
- :no_entry_sign: Makes breaking changes to the vxl/core/
- :no_entry_sign: Makes design changes to existing vxl/core/

## PR Description
New capabilities for CONTRIB BRL modules bpgl and bsgm.  
- disparity sign made consistent across BRL modules
- updated python modules accordingly
- removed redundant code
- additional convenience functions and parameters
- formatting corrections & additional comments

### CODE REVIEW

**Comment by drewgilliam** [comment]
@decrispell

File: contrib/brl/bseg/bsgm/bsgm_prob_pairwise_dsm.cxx [hunk 1]
```diff
@@ -13,13 +13,32 @@
 #include <bvgl/bvgl_k_nearest_neighbors_3d.h>
 #include <bpgl/algo/bpgl_heightmap_from_disparity.h>
 #include <bpgl/algo/bpgl_gridding.h>
-  
 void bsgm_prob_pairwise_dsm::compute_byte()
 {
+<<<<<<< HEAD
```
Comment (hjmjohnson) [inline contrib/brl/bseg/bsgm/bsgm_prob_pairwise_dsm.cxx]: There is a merge conflict here.

Comment (drewgilliam) [reply contrib/brl/bseg/bsgm/bsgm_prob_pairwise_dsm.cxx]: @hjmjohnson - the merge conflict was an oversight fixed in a later commit (in this same PR).  I updated the branch to eliminate the conflict from the commit history.

**Comment by drewgilliam** [comment]
@hjmjohnson - updated to eliminate merge conflict from commit history.

**Pull request merged**